gtfs-realtime/spec/en/reference.md

```diff
@@ -38,38 +38,61 @@ Fields labeled as **experimental** are subject to change and not yet formally ad
 
 ## Element Index
 
-*   [FeedMessage](#message-feedmessage)
-    *   [FeedHeader](#message-feedheader)
-        *   [Incrementality](#enum-incrementality)
-    *   [FeedEntity](#message-feedentity)
-        *   [TripUpdate](#message-tripupdate)
-            *   [TripDescriptor](#message-tripdescriptor)
-                *   [ScheduleRelationship](#enum-schedulerelationship-1)
-            *   [VehicleDescriptor](#message-vehicledescriptor)
-            *   [StopTimeUpdate](#message-stoptimeupdate)
-                *   [StopTimeEvent](#message-stoptimeevent)
-                *   [ScheduleRelationship](#enum-schedulerelationship)
-                *   [StopTimeProperties](#message-stoptimeproperties)
-            *   [TripProperties](#message-tripproperties)
-        *   [VehiclePosition](#message-vehicleposition)
-            *   [TripDescriptor](#message-tripdescriptor)
-                *   [ScheduleRelationship](#enum-schedulerelationship-1)
-            *   [VehicleDescriptor](#message-vehicledescriptor)
-            *   [Position](#message-position)
-            *   [VehicleStopStatus](#enum-vehiclestopstatus)
-            *   [CongestionLevel](#enum-congestionlevel)
-            *   [OccupancyStatus](#enum-occupancystatus)
-            *   [CarriageDetails](#message-carriagedetails)
-        *   [Alert](#message-alert)
-            *   [TimeRange](#message-timerange)
-            *   [EntitySelector](#message-entityselector)
-                *   [TripDescriptor](#message-tripdescriptor)
-                    *   [ScheduleRelationship](#enum-schedulerelationship-1)
-            *   [Cause](#enum-cause)
-            *   [Effect](#enum-effect)
-            *   [TranslatedString](#message-translatedstring)
-                *   [Translation](#message-translation)
-            *   [SeverityLevel](#enum-severitylevel)
+- [Elements](#elements)
```

Comment:
Are these changes to the table of contents intended?

Reply:
No I'll revert those, it's from my editor.